index.html

```diff
@@ -289,6 +289,36 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <section class="contact_form">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in <span>touch.</span>
+        </p>
+        <form action="#" class="form" method="get">
+          <input
+            name="Name"
+            id="Name"
+            type="text"
+            placeholder="FullName"
+            maxlength="30"
+          /><br />
+          <input
+            name="Email"
+            id="email"
+            type="text"
```

Comment:
- [ ] :wave: you missed out on `required` and the `Placeholder`  on the text. I suggest you implement them, this should easy. :clinking_glasses: